spec/requests/api/v1/merchants_request_spec.rb

```diff
@@ -17,7 +17,7 @@
 
       merchants["data"].each do |merchant|
         expect(merchant).to have_key("id")
-        expect(merchant["id"]).to be_a(Integer)
+        expect(merchant["id"]).to be_a(String)
```

Comment:
was having this same issue but I think I converted them back to integers as JSON turned them into strings. Good catch